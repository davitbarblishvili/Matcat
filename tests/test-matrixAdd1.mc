func main() int {
    matrix c; 
    c = [[1,2],[3,4]] + [[3,2],[5,4]];
    printm(c);

    return 0;
}