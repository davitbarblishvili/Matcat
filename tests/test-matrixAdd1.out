(
[4.00 4.00 ]
[8.00 8.00 ]
)